src/components/blog/BlogItem.js

```diff
@@ -133,108 +126,171 @@ class BlogItem extends Component {
-                                                                    <button className="float-right btn btn-primary"
-                                                                            type="submit">
-                                                                        Post
-                                                                    </button>
-                                                                </form>
-                                                            </div>
-                                                            : null
-                                                        }
+                            </div>
+                        </div>
 
-                                                        <div className="card" key={comment.id}>
-                                                            {comment.reply.map((reply) => (
-                                                                <div className="card" key={comment.id}>
-                                                                    <div className="card-body" key={reply.id}>
-                                                                        <h5 className="card-title">{reply.description}</h5>
-                                                                        <p className="card-text">By: {reply.owner}</p>
-                                                                        <h6 className="card-subtitle mb-2 text-muted">{(new Date(reply.created)).toString()}</h6>
-                                                                        {isAuthenticated && this.props.auth.user.username === reply.owner ?
-                                                                            <button className="btn btn-danger btn-sm"
-                                                                                    onClick={this.deleteComment(reply.id)}> Delete
-                                                                            </button>
-                                                                            : null
-                                                                        }
-                                                                    </div>
+                        {this.props.messages.vote ?
+                            <p className='mt-4 alert-danger'>{this.props.messages.vote}</p>
+                            : null}
+                        {this.props.blogItem.image ?
+                            <img className="mt-4 img-fluid rounded"
+                                 src={this.props.blogItem.image}
+                                 alt=""
+                            />
+                            :
+                            <img className="mt-4 img-fluid rounded"
+                                 src={defaultImage}
+                                 alt=""
+                            />
+                        }
+
+                        <br/><br/><br/>
```

Comment:
Avoid doing this, use one `<br/>` and add padding/margin on the elements before and after this

Reply:
It's been removed.